.gitignore

```diff
@@ -1,3 +1,4 @@
 dist/
 build/
 contrib/
+*.swp
```

Comment:
Unrelated?

Reply:
I didn't think it merits its own, separate pull request.